### DISCUSSION TITLE
📊 Use QuickSight for AWS cost and usage metrics

### DISCUSSION THREAD

**Discussion by bagg3rs** [ministryofjustice/analytical-platform#2640] [Firebreak Winter 23]
**Idea Description**
💰 Visualise the Cost and Usage reports across our accounts and track our progress when e.g. shutting down EKS clusters and other cost optimisation 

**Why Should We Do This**
🌱 We are very reactive to cost currently and there are many unknowns to how much things cost.
It could be a data product that could be used across other teams to understand and communicate cost to others where Cloud Platform is not used e.g. [cost by namespace](https://reports.cloud-platform.service.justice.gov.uk/costs_by_namespace)https://reports.cloud-platform.service.justice.gov.uk/costs_by_namespace
Also exploring the managed BI Tool that we already have available, doesn't require a Window VM running somewhere (PowerBI Gateway) and looks pretty nice, we also get to understand if its extremely painful to manage.

**Definition of Done**
- 📊 Dashboard showing costs and metrics from a AWS account
- Stretch DoD show all our accounts in one dashboard
- Stretch DoD use [Amazon Q](https://aws.amazon.com/quicksight/q/) to ask questions about cost etc
- Answer the question, is QS a better place for collating DORA metrics?

**Resources**
- https://catalog.workshops.aws/awscid/en-US
- https://www.wellarchitectedlabs.com/cloud-intelligence-dashboards/#foundational-dashboards

**What skills do we need?**
AWS / Terraform / QuickSight

**Comment by jasonBirchall** [comment]
👁️ Operations Engineering is watching with interest. This is a great idea, and we are also keen to explore something similar.
